apps/portal/components/pagesComponents/_singleImageScreen/SingleImagePageManageAreaTemplate.tsx

```
@@ -486,15 +488,15 @@ const SingleImagePageManageAreaTemplate: React.FC<Props> = ({
           <TabPanel selectedClassName={styles.selectedTabPanel}>
             <ImageActionsCard className="tw-rounded-t-none">
               <div>
-                Currently saved in
+                {t('page.image.currentlySavedIn')}
                 <strong className="tw-ml-1.5">{currentlySavedIn}</strong>
               </div>
               <div
                 className={styles.moveToFolder}
                 onClick={() => setShowFolderModal(true)}
               >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)